deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -57,51 +58,44 @@ class AbstractPropertyOutliers(SingleDatasetCheck):
         Two percentiles which define the IQR range
     iqr_scale: float, default: 1.5
         The scale to multiply the IQR range for the outliers detection
+    properties_precalculated: bool, default: False
```

Comment:
name is a bit weird. I suggest maybe "are_properties_precalculated" or just "precalculated_properties"

Reply:
@nirhutnik This parameter was removed, now removed from docstring as well